src/rules/color-named/index.js

```diff
@@ -55,6 +56,11 @@ export default function (expectation, options) {
       valueParser(decl.value).walk(node => {
         const { value, type, sourceIndex } = node
 
+        if (
+          optionsMatches(options, "ignore", "inside-function")
+          && type === "function"
+          && !colorFunctionNames.has(value.toLowerCase())) { return false }
```

Comment:
> Do you think this line is important? Right now I'm thinking it should be cut. I think if it ever actually came up it would be more confusing than helpful to the end user.

Sgtm. I was thinking of a particular edge-case where this option could be used with the `"always-where-possible"` primary option.  However, this seems unlikely.

@gaidarenko Can you please revert this conditional to it's original form. And then I think we're ready to merge.